clusterloader2/testing/density/config.yaml

```diff
@@ -3,6 +3,7 @@
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
 #Constants
+{{$RESOURCE_CONSTRAINTS_FILE := DefaultParam .RESOURCE_CONSTRAINTS_FILE "constraints_100.yaml"}}
```

Comment:
I don't think it should be default.
Default should be empty, and we should overwrite it for 100-node test only.

Reply:
I removed default value. This value will be provided with overrides.